serverless/containers/reference-content/configure-trigger-inputs.mdx

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
This page needs to be added to the menu via an edit to the `menu/navigation.json` file

Reply:
I added it (in the wrong PR, but it will be here when everything is merged)